vendor/bat-native-ads/src/bat/ads/internal/features/text_embedding_features.cc

```diff
@@ -0,0 +1,50 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "bat/ads/internal/features/text_embedding_features.h"
+
+#include "base/metrics/field_trial_params.h"
+
+namespace ads {
+namespace targeting {
+namespace features {
+
+namespace {
+
+constexpr char kFeatureName[] = "TextEmbedding";
+
+constexpr char kFieldTrialParameterPageEmbeddingsHistorySize[] =
+    "page_embeddings_history_size";
```

Comment:
nit: maybe `text_embeddings_history_size` for consistency?

Reply:
same for `kDefaultPageEmbeddingsHistorySize` => `kDefaultTextEmbeddingsHistorySize`

Reply:
Updated